### PR TITLE
migrate images from docker.io to quay.io

### PR DESCRIPTION
**Pull request by praveen4g0**
Migrated all images form docker.io to ubi8

### CODE REVIEW

**Review by ppitonak — CHANGES_REQUESTED** [review]
Use UBI8 images wherever possible.

**Comment by praveen4g0** [comment]
@ppitonak updated images to use ubi8 insted :wink:

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM golang:1.14
+FROM golang:latest
```
Comment (ppitonak) [inline Dockerfile:1]: I would like also this one changed. Can be done in new PR

**Review by ppitonak — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (praveen4g0) [reply Dockerfile:1]: Noted

**Pull request merged**